test/integration/object/validateDPObjectFactory.spec.js

```diff
@@ -356,6 +369,33 @@ describe('validateDPObjectFactory', () => {
     });
   });
 
+  describe('$action is DELETE', () => {
+    it('should validate against base DP object schema', () => {
+      dpObjectBaseSchema.additionalProperties = false;
+
+      delete rawDPObject.name;
+      rawDPObject.$action = DPObject.ACTIONS.DELETE;
+
+      const result = validateDPObject(rawDPObject, dpContract);
+
+      expect(validator.validate).to.be.calledOnceWith(dpObjectBaseSchema, rawDPObject);
+      expect(result.getErrors().length).to.be.equal(0);
+    });
+
+    it('should throw validation error if additional fields are defined', () => {
+      dpObjectBaseSchema.additionalProperties = false;
```

Comment:
Hm... it works only for this test because you set `dpObjectBaseSchema.additionalProperties = false;` here.